plugins/woocommerce/src/Admin/PluginsHelper.php

```diff
@@ -37,13 +37,18 @@ class PluginsHelper {
 	 *
 	 * @var bool
 	 */
-	public static $can_show_expiring_subs_notice = true;
+	public static $can_show_expiring_or_missing_subs_notice = true;
```

Comment:
this one variable serves as a flag whether a notice is already shown / not (so the other notices should not be shown)

back when we only have 2 notices the naming makes sense, however since currently we have 3 types of notices, maybe it's time to rename this? maybe something like `subscription_notice_already_shown`?

Reply:
Renamed in 0e9b92fecf. I also added a docbloc to better describe this as there is a clear priority of notification order.

Reply:
also if we use this name: `subscription_notice_already_shown` i realized that the boolean logic needs to be flipped

Reply:
Ah, thanks again, flipped in https://github.com/woocommerce/woocommerce/pull/51060/commits/8df6d945e4eadd6e328624caf3a0645fead0d110